packages/doenetml-worker-javascript/src/utils/dast/types.ts

```diff
@@ -0,0 +1,243 @@
+import { Position, RefResolution, Source } from "@doenet/doenetml-worker";
+
```

Comment:
This file is part of the bridge between DAST and the old-style serialized state?

Reply:
Yes, this file describes the serialized state, which is pretty close to the old-style serialized state, that I'm attempting to use as the target for dast than I usable in the old code without too many modifications.